boards/nxp/lpcxpresso55s69/Kconfig.lpcxpresso55s69

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_LPCXPRESSO55S69_LPC55S69_CPU0
+	bool
+	select SOC_LPC55S69_CPU0
+
+config BOARD_LPCXPRESSO55S69_LPC55S69_CPU0_NS
+	bool
+	select SOC_LPC55S69_CPU0
+
+config BOARD_LPCXPRESSO55S69_LPC55S69_CPU1
+	bool
+	select SOC_LPC55S69_CPU1
+
+config SOC_PART_NUMBER
+	default "LPC55S69JBD100" if BOARD_LPCXPRESSO55S69_LPC55S69_CPU0 || \
```

Comment:
We should not be setting `SOC_PART_NUMBER` here, we should select `SOC_PART_NUMBER_LPC55S69JBD100`

Reply:
Fixed